scilpy/utils/spatial_ops.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+
+def parallel_transport_streamline(streamline, nb_streamlines, radius, rng=None):
```

Comment:
After discussion with Arnaud, we would put this in tractogram.streamline_operations.

Can you also move the r(vec, theta) method into utils.util, and name it something like def rotation_around_vector_matrix?